tower_modules/tower_settings/tasks/main.yml

```diff
@@ -1,14 +1,36 @@
 - name: Set the value of AWX_PROOT_SHOW_PATHS
   tower_settings:
     name: "AWX_PROOT_SHOW_PATHS"
-    value: "'/var/lib/awx/projects/', '/tmp'"
+    value:
+      - '/var/lib/awx/projects/'
+      - '/tmp'
+  register: result
+
+- assert:
+    that:
+      - "result is changed"
+
+- name: Revert the value of AWX_PROOT_SHOW_PATHS to its default
+  tower_settings:
+    name: "AWX_PROOT_SHOW_PATHS"
+    value: []
   register: result
 
 - assert:
     that:
       - "result is changed"
 
 - name: Set the value of AWX_PROOT_BASE_PATH
+  tower_settings:
+    name: AWX_PROOT_BASE_PATH
+    value: "/tmp/newtmp"
```

Comment:
The default value of this option is `/tmp` so we are setting it to another value to ensure it was actually changed.